src/settings/DisplaySettings.ts

```diff
@@ -0,0 +1,6 @@
+export class DisplaySettings {
+    public static readonly CANVAS_MIN_MARGIN_PX: number = 20;
+    public static readonly MIN_ZOOM_PERCENTAGE: number = 100;
```

Comment:
takie pytanie bo mnie trochę ciekawi: łatwiej Ci operować procentami? nie lepiej założyć że `1` to widok 100%? nie wiem :D pytam :)

Reply:
Łatwiej mi :) jakoś tak mi to odpowiada